modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/PredefinedValuesTable.tsx

```diff
@@ -60,7 +60,7 @@ export default function PredefinedValuesTable({
 						<div className="lfr-object-web__predefined-values-table-input-method">
```

Comment:
change commit message to `User cannot change any system object action property via UI, except the label`